openmc_dagmc_wrapper/Tally.py

```diff
@@ -24,38 +24,49 @@ def tally_type(self):
 
     @tally_type.setter
     def tally_type(self, value):
-        output_options = (
-            [
-                "TBR",
-                "flux",
-                "heating",
-                "photon_heating",
-                "neutron_heating",
-                "neutron_flux",
-                "photon_flux",
-                "absorption",
-                "neutron_effective_dose",
-                "photon_effective_dose",
-                "neutron_fast_flux",
-                "photon_fast_flux",
-                "neutron_spectra",
-                "photon_spectra",
-            ]
-            + list(REACTION_MT.keys())
-            + list(REACTION_NAME.keys())
-        )
+        output_options = [
+            "TBR",
+            "neutron_flux",
+            "photon_flux",
+            "neutron_fast_flux",
+            "photon_fast_flux",
+            "photon_heating",
+            "neutron_heating",
+            "neutron_effective_dose",
+            "photon_effective_dose",
+            "neutron_spectra",
+            "photon_spectra",
+            None,
+        ]
         if value not in output_options:
-            raise ValueError(
-                "tally_type argument",
-                value,
-                "not allowed, the following options are supported",
-                output_options,
+            msg = (
+                f"tally_type argument {value} is not supported, the "
+                f"following options are supported {output_options}"
             )
+            openmc_supported_scores = (
+                list(REACTION_MT.keys())
+                + [str(mt) for mt in list(REACTION_MT.keys())]
+                + list(REACTION_NAME.keys())
+            )
+
+            if value in openmc_supported_scores:
+                msg = (
+                    msg + f"\n {value} is supported by native OpenMC scores "
+                    f"Try setting the Tally with scores=[{value}] instead "
+                    "of with tally_type"
+                )
+
+            raise ValueError(msg)
+
         self._tally_type = value
 
-    def set_score(self):
+    def set_score(self, **kwargs):
+        if "scores" in kwargs:
+            self.scores = kwargs["scores"]
+            if self.tally_type is not None:
+                raise ValueError("A score and a tally_type can not both be set")
```

Comment:
Maybe this should be in init? It would avoid having to bring kwargs in set_score

Reply:
I've moved this to the init as suggested, thanks for the tip